src/Castle.Windsor.Extensions.DependencyInjection/README.md

```diff
@@ -1,25 +1,39 @@
-# windsor-service-provider
-Service Provider using Castle Windsor container for ASP.NET Core 3.x. Fully replaces ASP.NET Core Default Service Provider (container). Supports both registering services
-* using `IServiceCollection` (`Startup.ConfigureServices`) - these get translated to Castle Windsor registrations
-* `Startup.ConfigureContainer`
-    ```
-    public void ConfigureContainer (IWindsorContainer container)
-    {
-        container.Install(new MyInstaller());
-    }
-    ```
+# .NET Core 3.0 Dependency injection extension
```

Comment:
Can you move and rename this README.md file into the root docs directory so all the docs are together even though they are no longer published to the web site.

Once this package is in I'll clean up the facilities page to link to this and mark the existing packages for ASP.NET Core 2.0 as obsolete.